ted_sws/metadata_normaliser/services/metadata_normalizer.py

```diff
@@ -68,6 +71,34 @@ def normalise_metadata(self):
         self.notice.set_normalised_metadata(normalised_metadata)
 
 
+def filter_df_by_variables(df: pd.DataFrame, column_one_name: str, var_one: str, column_two_name: str = None,
```

Comment:
I appreciate the intention of generalising. Well done! However, In computer science school, they will teach to never use increments, in the function parameters of the type 
`column_one_name`, `var_one`, `..._two`, `..._three` etc ...

Perhaps, this is a "good enough" solution for now. The properly generalised solution shall still come to us (one day).